pipes/new_signal_minute.pipe

```diff
@@ -0,0 +1,24 @@
+DESCRIPTION >
+	Round Time Down to minute
+
+
+NODE signal_minute_0
+SQL >
+
+    SELECT 
+        toStartOfMinute(time) AS time,
+        network,
+        COALESCE(NULLIf(asset, ''), signal) AS signal,
```

Comment:
should this be `as asset` so the pipe is compatible with the existing target data source? Otherwise a full repopulation is needed